.rubocop.yml

```diff
@@ -0,0 +1,188 @@
+
+Layout/SpaceBeforeSemicolon:
+  Description: "No spaces before semicolons."
+  Enabled: true
+
+Layout/SpaceAroundOperators:
+  Description: "Use a single space around operators."
+  StyleGuide: "#spaces-operators"
+  Enabled: true
+
+Layout/Tab:
+  Description: "No hard tabs."
+  StyleGuide: "#spaces-indentation"
+  Enabled: true
+
+Layout/TrailingBlankLines:
+  Description: "Checks trailing blank lines and final newline."
+  StyleGuide: '#newline-eof'
+  Enabled: true
+
+Layout/TrailingWhitespace:
+  Description: "Avoid trailing whitespace."
+  StyleGuide: "#no-trailing-whitespace"
+  Enabled: true
+
+#################### Naming ##############################
+
+Naming/ClassAndModuleCamelCase:
+  Description: "Use CamelCase for classes and modules."
+  StyleGuide: "#camelcase-classes"
+  Enabled: true
+
+Naming/ConstantName:
+  Description: "Constants should use SCREAMING_SNAKE_CASE."
+  StyleGuide: "#screaming-snake-case"
+  Enabled: true
+
+#################### Style ###############################
+
+Style/ClassVars:
+  Description: 'Avoid the use of class variables.'
+  StyleGuide: '#no-class-vars'
+  Enabled: true
+
+Style/DefWithParentheses:
+  Description: "Use def with parentheses when there are arguments."
+  StyleGuide: "#method-parens"
+  Enabled: true
+
+Style/Encoding:
+  Description: "Use UTF-8 as the source file encoding."
+  StyleGuide: "#utf-8"
+  Enabled: true
+
+# TODO
+Style/FrozenStringLiteralComment:
+  Description: "Add frozen_string_literal comment to the top of files"
+  Include:
+  - "app/**/*.rb"
+  - "lib/**/*.rb"
+  Enabled: false
+
+Style/HashSyntax:
+  Description: "Prefer Ruby 1.9 hash syntax { a: 1, b: 2 } over 1.8 syntax"
+  EnforcedStyle: ruby19
+
+Style/IfInsideElse:
+  Description: "Finds if nodes inside else, which can be converted to elsif."
+  Enabled: true
+
+Style/MethodCallWithoutArgsParentheses:
+  Description: "Do not use parentheses for method calls with no arguments."
+  StyleGuide: "#method-invocation-parens"
+  Enabled: true
+
+Style/MethodDefParentheses:
+  Description: >-
+                 Checks if the method definitions have or don't have
+                 parentheses.
+  StyleGuide: "#method-parens"
+  Enabled: true
+
+Style/RedundantReturn:
+  Description: "Don't use return where it's not required."
+  Enabled: true
+
+Style/RedundantSelf:
+  Description: "Don't use self where it's not needed."
+  Enabled: true
+
+Style/TrailingCommaInArguments:
+  Description: "Checks for trailing comma in argument lists."
+  StyleGuide: "#no-trailing-params-comma"
+  Enabled: true
+
+#################### Metrics ###############################
+
+# TODO
+Metrics/LineLength:
+  Description: "Limit lines to 80 characters."
+  StyleGuide: "#80-character-limits"
+  Enabled: false
+
+
+#################### Bundler ###############################
+
+Bundler/DuplicatedGem:
+  Description: "Checks for duplicate gem entries in Gemfile."
+  Enabled: true
+  Include:
+    - "**/Gemfile"
+    - "**/gems.rb"
```

Comment:
👍  Love that we cover this already 👯

Reply:
I had to google what `gems.rb` is all about. Good stuff, looking forward to it!